testsuite/MDAnalysisTests/topology/base.py

```diff
@@ -100,3 +90,12 @@ def test_creates_universe(self, filename):
         """Check that Universe works with this Parser"""
         u = mda.Universe(filename)
         assert isinstance(u, mda.Universe)
+
+    def test_guessed_attributes(self, filename):
+        """check that the universe created with certain parser have the same
+        guessed attributes as  when it was guessed inside the parser"""
+        u = mda.Universe(filename)
+        u_guessed_attrs = [attr.attrname for attr in u._topology.guessed_attributes]
+        for attr in self.guessed_attrs:
+            assert hasattr(u.atoms, attr)
+            assert attr in u_guessed_attrs
```

Comment:
```suggestion
            assert attr in u_guessed_attrs

```

Suggesting to add an empty line